### ISSUE TITLE
Show evaluated output without console.log

### ISSUE THREAD

**Issue by webdevian** [vsaravind007/nodeScratchpad#9]
It would be good if it followed the behaviour of the node repl and dev tools console and just evaluated whatever code you put in and printed the output. 

There perhaps would need to be a separate output box for console.log output if it is used.

![screenshot 2018-05-24 09 01 26](https://user-images.githubusercontent.com/5087745/40472425-5e9fa6d6-5f31-11e8-89a0-f41a90ca3c30.png)

**Comment by vsaravind007** [comment]
Makes sense to replicate the behaviour of Node REPL, perhaps a switch to toggle between REPL mode & current mode.

**Comment by webdevian** [comment]
Yeah. I know for quick JS snippets I normally go to the repl for (i.e. what happens when I cast _x_ to type of _y_) not having to include a console.log around it is useful
